packages/modelviewer.dev/data/docs.json

```diff
@@ -548,7 +548,7 @@
       {
         "name": "hotspot-*",
         "htmlName": "hotspot",
-        "description": "Any child element under <span class=\"attribute\">&lt;model-viewer&gt;</span> with a slot name starting with \"hotspot\" will be aligned with the 3D model using its <span class=\"attribute\">data-position</span> and <span class=\"attribute\">data-normal</span> attributes. See the <a href=\"../examples/annotations\">annotations example</a> for details."
```

Comment:
Why did you take these spans away?

Reply:
they aren't direct model-viewer attributes, I can add them back if you think it makes more sense to keep them?

Reply:
yeah, probably so; they are attributes defined by model-viewer. And besides, that class name is a bit misleading; it's really anything we want to look like code.

Reply:
yeah... but also, if everything looks like code, then the helpfulness also goes down and it can get confusing when viewing it, i.e. it would be nice if the code on the site signified something we created

Reply:
Good point; I don't have a strong opinion on it. But these are at least attributes we created.

Reply:
gotcha, and yeah maybe there should be quotation marks or a different style for other code that isn't ours -- just something to think about